README.md

```diff
@@ -1,55 +1,40 @@
-# DevExtreme Reactive
+# DevExtreme Reactive &middot; [![CircleCI](https://img.shields.io/circleci/project/github/DevExpress/devextreme-reactive/master.svg)](https://circleci.com/gh/DevExpress/devextreme-reactive) [![NPM](https://img.shields.io/npm/v/@devexpress/dx-core.svg)](https://www.npmjs.com/package/@devexpress/dx-core)
 
-[![CircleCI](https://img.shields.io/circleci/project/github/DevExpress/devextreme-reactive/master.svg)](https://circleci.com/gh/DevExpress/devextreme-reactive)
-[![NPM](https://img.shields.io/npm/v/@devexpress/dx-core.svg)](https://www.npmjs.com/package/@devexpress/dx-core)
+DevExtreme Reactive is a set of business components for React and Vue that deeply integrates with popular Bootstrap and Material Design UI libraries.
 
-## Business Components for React and Vue
-
-### Main Features
+### Common Features
 
 - Composable and extendable plugin-based architecture
 - 100% Native React and Vue (no jQuery or other dependencies)
 - High performance by using Vue and React best practicies
+- Native Bootstrap rendering with seamless theming
 
-## Business Components for React
-
-[Website](https://devexpress.github.io/devextreme-reactive/react/)
+## [React Components](https://devexpress.github.io/devextreme-reactive/react/)
 
-### Main React Features
+### Common Features
 
 - [Material UI](https://github.com/callemall/material-ui), [Bootstrap 4](http://reactstrap.github.io) and [Bootstrap 3](https://github.com/react-bootstrap/react-bootstrap) integration with seamless theming
 - Controlled (stateless) and uncontrolled (stateful) modes
 - [Redux](https://github.com/reactjs/redux/) integration with state persistence and time-traveling
 
-### React Grid
-
-[Website](https://devexpress.github.io/devextreme-reactive/react/grid/)
-|
-[Demos](https://devexpress.github.io/devextreme-reactive/react/grid/demos/)
-|
-[Docs](https://devexpress.github.io/devextreme-reactive/react/grid/docs/)
-
-### React Chart
+### React Data Grid
 
-Project status: **CTP**
+[Website](https://devexpress.github.io/devextreme-reactive/react/grid/) | [Demos](https://devexpress.github.io/devextreme-reactive/react/grid/demos/) | [Docs](https://devexpress.github.io/devextreme-reactive/react/grid/docs/)
 
-[Website](https://devexpress.github.io/devextreme-reactive/react/chart/)
-|
-[Docs](https://devexpress.github.io/devextreme-reactive/react/chart/docs/)
+### React Chart (CTP)
 
+[Website](https://devexpress.github.io/devextreme-reactive/react/chart/) | [Docs](https://devexpress.github.io/devextreme-reactive/react/chart/docs/)
 
-## Business Components for Vue
+*Note: there is an [alternative project](https://github.com/DevExpress/devextreme-react) providing 65+ React components. Take a look at the comparison [blog post](https://community.devexpress.com/blogs/oliver/archive/2018/04/20/devextreme-new-react-wrappers-vs-native-react-components.aspx) to learn if it meets your business needs better.*
```

Comment:
*You can also use the [alternative project](https://github.com/DevExpress/devextreme-react) with 65+ React components. Refer to the comparison [blog post](https://community.devexpress.com/blogs/oliver/archive/2018/04/20/devextreme-new-react-wrappers-vs-native-react-components.aspx) for more information.*